src/main/java/org/junit/rules/ExpectedException.java

```diff
@@ -14,7 +14,10 @@
 
 /**
  * The {@code ExpectedException} rule allows you to verify that your code
- * throws a specific exception.
+ * throws a specific exception. Note that {@link org.junit.Assert#assertThrows Assert.assertThrows}
```

Comment:
Link tags without parameters do work, but only if there are no overloads. Since that would be a fragile state of affairs, I've added the parameters. I also followed your other suggestions, except for putting the new text at the end. If we're trying to encourage people to move to `assertThrows` then we probably don't want to bury that suggestion at the end of the text? Putting it at the top feels more like the effect we would get with `@deprecated`, though I admit it is a bit clunky.